local-testing-support/keycloak-docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Build a container for running the 1.9.1 version of Keycloak.
+# Assume Keycloak folder mounted as a volume in /srv/keycloak.
+#
+# Build with:
+# docker build -t keycloak-1.9.1 .
```

Comment:
Seems like this wont work.  we commented out the docker build for keycloak.  Is this because we are testing a different way now?

Reply:
Comment is command to actually build the Docker image. This is the config file used by the build command.